lib/components/src/controls/Date.tsx

```diff
@@ -99,15 +99,15 @@ export const DateControl: FC<DateProps> = ({ name, value, onChange, onFocus, onB
         type="date"
         max="9999-12-31" // I do this because of a rendering bug in chrome
         ref={dateRef as RefObject<HTMLInputElement>}
-        id={`${name}date`}
-        name={`${name}date`}
+        id={`${name}-date`}
+        name={`${name}-date`}
```

Comment:
I've noticed that `Checkbox`'s and `Radio`'s `id` is formed as `${name}-${key}`, so I decided change `id` and `name` here for consistency. I'll revert it if you tell me.